### ISSUE TITLE
Content type bulk update isn't changing content type _from_ book

### ISSUE THREAD

**Issue by andrewjbtw** [sul-dlss/argo#2694]
**Describe the bug**
When using the bulk update to change the content type for a set of druids, attempts to change the content type **from** book to another content type are not working.

- if the object is in a registered state, the content type remains book.
- if the object is "opened", the content type remains book, though the resource type is changed

**User Impact**
Users have to change the content type for each item individually and cannot use the bulk update.

**To Reproduce**
1. Go to a book object like https://argo-stage.stanford.edu/view/druid:zy005bt8666
2. Use the bulk update to change the content type
https://argo-stage.stanford.edu/report/bulk?q=zy005bt8666&search_field=text&utf8=%E2%9C%93
3. Choose these settings
- old content type: book
- old resource type: page
- new content type: image
- new content type: image
4. See the bulk update report success
5. Go back to the druid and see that content type is still book

**Expected behavior**
The bulk change of content type should change to the new content type

**Additional context**
This seems to work for non-book content types. It also works on the individual item blue button menu.

**Comment by jcoyne** [comment]
@andrewjbtw @aaron-collier  this was introduced by #2562.  The Batch version of this doesn't have awareness of reading direction for the book in the "Old content type" field.  This is populated by looking at the `content_type_ssim` solr field for the list of items in the batch.  Since that field doesn't have reading direction, it'll always be wrong it its a book. 

I think we have a couple of different routes we could choose from here:

1. Decouple reading direction from content type on the "set content type" menu
1. Index the content type with reading direction
1. Ignore what is in the batch and present all possible content types and rely on the user to know what is in the batch. 
1. If "book" is present in the solr query result, split it into the two directions and leave it up to the user to know which one is in the batch.

**Comment by peetucket** [comment]
I think this may have to do with `content_type_should_change?` returning `false` when it should return `true` at https://github.com/sul-dlss/argo/blob/main/app/controllers/content_types_controller.rb#L96-L98

I'm pretty sure this will fix it: https://github.com/sul-dlss/argo/pull/2704/files

It uses the method that does a computation for the old content type, which looks at the reading order in structural metadata.

**Comment by peetucket** [comment]
Edit... the tests fails so maybe not, though I think it's on the right path.  It won't populate the "old content type" drop down correctly still for the reasons Justin noted above, but it should change them to the new value you selected if it is different than what is there.

**Comment by andrewjbtw** [comment]
This has always seemed like an odd bulk action to me. I realize I can't speak for all users, but if I make a list of druids where I want to change the content type, then I would expect the content type to be changed for all druids on my list. I wouldn't want the bulk action to skip something on the list because of the value for the "old" content type. If I didn't want it changed, I wouldn't put it on the list. So requiring the old content type in this context doesn't seem necessary, and in the blue button content type menu for a single object you don't have to supply the old value (though it is pre-filled and displayed in the modal).

Resource types are different because a content type can include multiple resource types, so I understand why the old value is needed there. But an object can only have one content type so I would interpret a change of content type to be a change to apply regardless of what the previous value is there. I'm pretty sure this bulk action does make it possible to create combinations of content and resource types that do not work in the viewer, but I think that's always been the case.

**Comment by jcoyne** [comment]
@andrewjbtw I'll take a look tomorrow with that simplification in mind. That ought to be enough to unblock this.

**Comment by peetucket** [comment]
That makes sense Andrew. I didn't understand why the old types were needed for bulk updates either and not requiring them would make it all easier.
